e2e/utilities/repo-client/apis/repo-api.ts

```diff
@@ -47,6 +46,9 @@ export abstract class RepoApi {
 
     protected handleError(message: string, response: any) {
       console.log(`\n--- ${message} error :`);
+      console.log('\t>>> username: ', this.username);
+      console.log('\t>>> password: ', this.password);
```

Comment:
This would print the password in the travis log. Not sure it's a good idea from a security point of view